src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -4344,8 +4344,8 @@ Object {
   "font-chart-detail-size": "12px",
   "font-display-label-weight": "700",
   "font-expandable-heading-size": "16px",
-  "font-family-base": "\\"Open Sans\\", \\"Helvetica Neue\\", Roboto, Arial, sans-serif",
-  "font-family-monospace": "Monaco, Menlo, Consolas, \\"Courier Prime\\", Courier, \\"Courier New\\", monospace",
+  "font-family-base": ""Open Sans", "Helvetica Neue", Roboto, Arial, sans-serif",
```

Comment:
Could this be an issue?  I see this is a snapshot but does this snapshot reflect that the css styling could be broken being applied via the theme?  Also with below

Reply:
I think this is fine as of being the updated snapshot representation only. Should there be a real regression in the future the test will capture it

Reply:
I see no issues here. Newer Jest changed their snapshot format, quotes do not need escaping here, that's all: https://jestjs.io/docs/upgrading-to-jest29#snapshot-format